nullaway/build.gradle

```diff
@@ -76,3 +76,22 @@ test {
 }
 
 apply from: rootProject.file("gradle/gradle-mvn-push.gradle")
+
+install {
+  repositories.mavenInstaller {
+    pom.scopeMappings.mappings.remove(project.configurations.compile)
```

Comment:
Move the common three lines to a def like

```
def configurePom(deployer) {
    deployer.pom.....
}
```

Then call it on `repositories.mavenDeployer` and `repositories. mavenInstaller`